source/index.html.md

```diff
@@ -2101,7 +2104,8 @@ This endpoint is use to returns a list of subscription entries.
       "tickets_count": 1000,
       "ticket_price": "25.00",
       "currency": "NOK",
-      "cover_url": null
+      "cover_url": null,
+      "repeat_every": 604800
```

Comment:
@JoelShinness-FFATG Can we add somewhere to the doc what the metric is?  I assume it is seconds?

Reply:
It is seconds.  I didn't see a place to add the note other than "number", but I'll take another look.